run-tests.php

```diff
@@ -3266,10 +3275,30 @@ function show_result(
     string $extra = '',
     ?array $temp_filenames = null
 ): void {
-    global $temp_target, $temp_urlbase, $line_length, $SHOW_ONLY_GROUPS;
+    global $SHOW_ONLY_GROUPS, $colorize;
 
     if (!$SHOW_ONLY_GROUPS || in_array($result, $SHOW_ONLY_GROUPS)) {
-        echo "$result $tested [$tested_file] $extra\n";
+        if ($colorize) {
+            /* Use ANSI escape codes for coloring test result */
+            switch ( $result ) {
+                case 'PASS': // Light Green
+                    $color = "\e[1;32m{$result}\e[0m"; break;
+                case 'SKIP': // Light Gray
+                    $color = "\e[0;37m{$result}\e[0m"; break;
```

Comment:
This ends up being pretty much the same as the normal text color :/ Maybe use yellow or orange?

Reply:
Basically, let this fall through the the default case that also handles XFAIL. XFAIL and SKIP seems pretty similar.